packages/web-app-admin-settings/src/components/Users/GroupsModal.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <oc-modal
+    :title="title"
+    :message="message"
+    :button-cancel-text="$gettext('Cancel')"
+    :button-confirm-text="$gettext('Confirm')"
+    :button-confirm-disabled="!selectedOptions.length"
+    @cancel="$emit('cancel')"
+    @confirm="$emit('confirm', { users, groups: selectedOptions })"
+  >
+    <template #content>
+      <oc-select
```

Comment:
Possible to reuse the `GroupSelect.vue` component instead? Right now the two group assignments (batch or single user in sidebar) look very different. Would prefer to have them look the same.